src/main/java/com/smartparkinglot/backendservice/services/reservationservice/ReservationService.java

```diff
@@ -0,0 +1,15 @@
+package com.smartparkinglot.backendservice.services.reservationservice;
+
+import com.smartparkinglot.backendservice.domain.Reservation;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+
+public interface ReservationService {
+    ResponseEntity<List<Reservation>> listAll();
```

Comment:
If you did not see my comments at other pull request. I explained what is the best practice and what is my suggestion to you. Same suggestion includes this PR too. [Other PR](https://github.com/bitirme-projesi-beu/backend-service/pull/1)